test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -431,5 +431,92 @@ public void Tokenize()
             Assert.True(type.IsVector && !type.IsKnownSizeVector && type.ItemType.IsKey);
             Assert.True(type.ItemType.AsKey.RawKind == DataKind.U2);
         }
+
+        [Fact]
+        public void NormalizeTextAndRemoveStopWords()
+        {
+            var env = new ConsoleEnvironment(seed: 0);
+            var dataPath = GetDataPath("wikipedia-detox-250-line-data.tsv");
+            var reader = TextLoader.CreateReader(env, ctx => (
+                    label: ctx.LoadBool(0),
+                    text: ctx.LoadText(1)), hasHeader: true);
+            var dataSource = new MultiFileSource(dataPath);
+            var data = reader.Read(dataSource);
+
+            var est = data.MakeNewEstimator()
+                .Append(r => (
+                    r.label,
+                    normalized_text: r.text.Normalize(),
```

Comment:
>Normalize [](start = 44, length = 9)

NormalizeText? Because we have Normalize with a completely different meaning, working on floats